src/MicrosoftTeams.ts

```diff
@@ -175,8 +176,9 @@ namespace microsoftTeams {
     /**
      * Initializes the library. This must be called before any other SDK calls
      * but after the frame is loaded successfully.
+     * @param {string} [appCorrelationId] Optional unique ID for use in correlating telemetry data.
      */
-    export function initialize(): void {
+    export function initialize(appCorrelationId?: string): void {
```

Comment:
I am considering removing the option to provide a correlationId. I think we have 3 options here and I am still trying to evaluate which one is the best:
1. Have the developer provide a correlation ID on initialization and simply pass it back to the Teams client.
2. Always generate a unique ID on initialization and pass it back to the Teams client.
3. Have the Teams client either generate a unique ID or pass its session ID to the SDK.

Reply:
I need to also think through the implications for this when the parent isn't Teams but the tab content/configuration page instead. This happens when the SDK is loaded inside the authentication popup and the app is running in our web client.